app/jobs/create_rss_notifications.rb

```diff
@@ -0,0 +1,16 @@
+class CreateRssNotifications < ApplicationJob
+  after_perform do |job|
+    self.class.set(:wait => 12.hours).perform_later(job.arguments.first)
+  end
+
+  def perform(options = {})
+    # Defaults to theforeman.org blog RSS
+    UINotifications::RssNotificationsChecker.new(options).deliver!
+  end
+
+  rescue_from(StandardError) do |error|
+    Foreman::Logging.logger('background').error(
+      'RSS notification checker: '\
```

Comment:
whats the reason for the multiline? I assume its not multiline in the log?

Reply:
Being able to stick to 80 chars & read stuff nicely in vim/tmux 😄